modules/apps/site-initializer/site-initializer-testray/extra/remote-app/src/pages/Standalone/Teams/useTeamActions.ts

```diff
@@ -30,11 +34,22 @@ const useTeamActions = () => {
 			permission: 'UPDATE',
 		},
 		{
-			action: ({id}: TestrayTeam, mutate) =>
-				deleteResource(`/teams/${id}`)
-					?.then(() => removeItemFromList(mutate, id))
-					.then(modal.onSave)
-					.catch(modal.onError),
+			action: ({id}: TestrayTeam, mutate) => {
+				getTeamsComponentsQuery(id)
+					.then((response) => {
+						if (response?.items?.length) {
+							throw new Error(
+								'The team cannot be deleted because it has associated components.'
```

Comment:
Use i18n to translate this

```suggestion
								'The team cannot be deleted because it has associated components.'
```